Frends.MySQL.ExecuteProcedure/Frends.MySQL.ExecuteProcedure/ExecuteProcedure.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Frends.MySQL.ExecuteProcedure.Definitions;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Dynamic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Frends.MySQL.ExecuteProcedure;
+
+/// <summary>
+/// Task class.
+/// </summary>
+public class MySQL
+{
+    /// <summary>
+    /// Execute a stored procedure to MySQL.
+    /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.MySQL.ExecuteProcedure)
+    /// </summary>
+    /// <param name="input"></param>
+    /// <param name="options"></param>
+    /// <param name="cancellationToken"/>
+    /// <returns>Object { int AffectedRows }</returns>
+    public static async Task<Result> ExecuteProcedure(
+        [PropertyTab] Input input,
+        [PropertyTab] Options options,
+        CancellationToken cancellationToken
+    )
+    {
+        if(string.IsNullOrWhiteSpace(input.ConnectionString)) throw new ArgumentNullException("Connection string cannot be empty.");
```

Comment:
If you want to throw `ArgumentNullException`, you should also define the argument in the error which was null. In this case, the correct syntax would be `throw new ArgumentNullException("Connection string cannot be empty.", nameof(input.ConnectionString))`.